source/_components/sensor.zabbix.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: "Zabbix Sensor"
+description: "Instructions how to integrate Zabbix Triggers sensors within Home Assistant."
+date: 2016-12-13 22:57
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: 
+ha_category: Sensor
+ha_release: 0.35
```

Comment:
Will need to be updated before being merged. Can wait until parent PR is merged to get the correct release number.